### ISSUE TITLE
Add a changelog

### ISSUE THREAD

**Issue by bradengroom** [confluentinc/kafka-connect-bigquery#90]
This project is great, but the lack of changelog makes it difficult to know what to expect when upgrading to the latest version.

**Comment by raphaelauv** [comment]
yes the changelog in the repo and on confluent website is not up to date

https://docs.confluent.io/kafka-connect-bigquery/current/changelog.html
